### PR TITLE
Fix/delete mocks

### PR DESCRIPTION
**Pull request by Barylo**
Deleted all mocks

### CODE REVIEW

**Review by VitaliiDenysiuk — APPROVED** [review]
Lgtm. But better delete comments

**Review by valeriisuvyk — APPROVED** [review]
LGTM

**Review by DavidKorystov — APPROVED** [review]

File: src/containers/CommentsList.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+// import { tweetCommentsData } from "../MOCKS/singleTweetPage";
 
 export const CommentsList: React.FC = () => {
   return (
     <div>
-      {tweetCommentsData.map((comment) => (
+      {/* {tweetCommentsData.map((comment) => (
         <SingleTweetComment key={comment.id} {...comment} />
-      ))}
+      ))} */}
```
Comment (valeriisuvyk) [inline src/containers/CommentsList.tsx]: You should to delete there comments

**Review by valeriisuvyk — COMMENTED** [review]
You should

**Review by KirillSerg — APPROVED** [review]
lgtm

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**